public/javascripts/index.js

```diff
@@ -41,17 +28,39 @@ const Letter = React.createClass({
 	    </div>
 	)
     }
-})
+});
 
-const Alphabet = React.createClass({
-    render() {
+var CapitalAButton = React.createClass({
+    showCapitalA: function(){
+	ReactDOM.render(<Letter />, document.querySelector("#letter"))
+    },
+    
+    render: function() {
 	return (
-	    <div id="a">
-	    <ButtonChoices />
-	    <Letter />
+	    <div id="capital-button" onClick={this.showCapitalA}>
+	    A
 	    </div>
-	);
-    },
+	)
+    }
+
 });
 
-ReactDOM.render(<Alphabet />, document.querySelector('#container'));
+var ButtonChoices = React.createClass({
+    render: function() {
+	return (
+	    <div id="buttons">
+	    <div id="button-group">
+	    <CapitalAButton />
+	    </div>
+	    </div>
+	)
+    }
+});
+
+
+ReactDOM.render(
+    <div id="a">
+    <ButtonChoices />
+    </div>,
+    document.querySelector('#container')
+);
```

Comment:
the top level should have a single component that is rendered with `ReactDOM.render`

all other components should be inside that 💃